mix.exs

```diff
@@ -39,21 +40,36 @@ defmodule Membrane.RTMP.Mixfile do
     [
       {:membrane_core, "~> 0.10.0"},
       {:unifex, "~> 1.0"},
-      {:ex_doc, "~> 0.28", only: :dev, runtime: false},
-      {:dialyxir, "~> 1.1", only: :dev, runtime: false},
-      {:credo, "~> 1.5", only: :dev, runtime: false},
-      {:membrane_hackney_plugin, "~> 0.8.0", only: :test},
       {:membrane_h264_ffmpeg_plugin, "~> 0.21.1"},
       {:membrane_aac_plugin, "~> 0.12.1"},
+      {:membrane_mp4_plugin, "~> 0.15.0"},
+      # testing
+      {:membrane_hackney_plugin, "~> 0.8.0", only: :test},
       {:ffmpex, "~> 0.7", only: :test},
-      {:membrane_mp4_plugin, "~> 0.15.0"}
+      # development
+      {:ex_doc, "~> 0.28", only: :dev, runtime: false},
+      {:dialyxir, "~> 1.1", only: :dev, runtime: false},
+      {:credo, "~> 1.5", only: :dev, runtime: false}
```

Comment:
I know it won't do anything, but given how credo 1.5 behaves with elixir 1.13, I think I would...
```suggestion
      {:credo, "~> 1.6", only: :dev, runtime: false}
```